# from camera frame to body frame
body_T_cam0:
  rows: 4
  cols: 4
  dt: d
  data:
    [
      1.8294689711463086e-02,
      2.4202947830259766e-02,
      9.9953965486352137e-01,
      3.4136692089063315e-02,
      -9.9962082400604302e-01,
      2.1019599407476885e-02,
      1.7787204789652095e-02,
      3.4560300051745159e-02,
      -2.0579420347548963e-02,
      -9.9948606481384983e-01,
      2.4578317702478425e-02,
      4.4924675289851203e-02,
      0.,
      0.,
      0.,
      1.,
    ]
